packages/react-storage/package.json

```diff
@@ -71,7 +71,7 @@
       "name": "createStorageBrowser",
       "path": "dist/esm/browser.mjs",
       "import": "{ createStorageBrowser }",
-      "limit": "18.55 kB",
+      "limit": "19.10 kB",
```

Comment:
We are wasting so much time on this, should we just set it to a high number and revisit when we launch?

Reply:
Yea, not opposed to setting this to something reasonable. I think this is useful for catching big deviations, but we're constantly updating it.